src/aics-image-viewer/components/App/index.tsx

```diff
@@ -128,6 +134,8 @@ const DEFAULT_CHANNEL_STATE: ChannelState = {
   isovalue: 128,
   opacity: 1.0,
   color: [226, 205, 179] as ColorArray,
+  useControlPoints: false,
+  ramp: [0, TFEDITOR_MAX_BIN],
```

Comment:
Not sure if this would make anything cleaner but maybe we could have the control points default to the equivalent of [[0, 0], [TFEDITOR_MAX_BIN, 1]]

Reply:
Also, rather than a bin index, we probably should think of the "x" domain of the transfer function (aka LUT) as intensities.  It's a function of input intensity to output intensity. 

If that incurs a crazy amount of changes then we should save it for a different PR and possibly wait for the full dynamic range modifications. ( also related to putting intensity numbers along the x axis in the UI.)